Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/rust/nestedmodules.dfy

```diff
@@ -0,0 +1,33 @@
+// NONUNIFORM: Temporary development of the Rust compiler
+// RUN: %baredafny run --target=rs "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+// RUN: %OutputCheck --file-to-check "%S/nestedmodules-rust/src/nestedmodules.rs" "%S/nestedmodules.check"
```

Comment:
Deserves a comment as to what the CHECK pattern is checking

Reply:
```
// Checks that, despite module B being prefixed with A, there will be a "mod B" somewhere
// and not an encoding like "mod A_B".
// This module will automatically be in mod A and referred to as "A::B"
```